org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -964,15 +964,25 @@ private void loadPrintData (PrintData pd, MPrintFormat format)
 											valueString = DisplayType.getDateFormat(pdc.getDisplayType(), m_language).format(value);
 										if (format.getTableFormat().isPrintFunctionSymbols())		//	Translate Sum, etc.
 											valueString	+= PrintDataFunction.getFunctionSymbol(functions[f]);
+										if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f])) {
+											Serializable groupValue = m_group.getValue(pdc.getAD_PrintFormatItem_ID(), 
+													pdc.getAD_PrintFormatItem_ID(), functions[f]);
+											if(groupValue instanceof Timestamp)
+												groupValue = DisplayType.getDateFormat(pdc.getDisplayType()).format((Timestamp)groupValue);
+											if(groupValue != null)
+												valueString += ": " + groupValue;
+										}
 										pd.addNode(new PrintDataElement(pdc.getAD_PrintFormatItem_ID(), pdc.getColumnName(),
 											valueString, DisplayType.String, false, pdc.isPageBreak(), pdc.getFormatPattern()));
 									}
 									else if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f]))
 									{
+										Serializable groupValue = m_group.getValue(group_pdc.getAD_PrintFormatItem_ID(), 
+												pdc.getAD_PrintFormatItem_ID(), functions[f]);
+										if(groupValue instanceof Timestamp)
+											groupValue = DisplayType.getDateFormat(pdc.getDisplayType()).format((Timestamp)groupValue);
 										pd.addNode(new PrintDataElement(pdc.getAD_PrintFormatItem_ID(), pdc.getColumnName(),
-											m_group.getValue(group_pdc.getAD_PrintFormatItem_ID(), 
-												pdc.getAD_PrintFormatItem_ID(), functions[f]), 
-											PrintDataFunction.getFunctionDisplayType(functions[f], pdc.getDisplayType()), 
+												groupValue, PrintDataFunction.getFunctionDisplayType(functions[f], pdc.getDisplayType()), 
 												false, pdc.isPageBreak(), pdc.getFormatPattern()));
```

Comment:
the change for this block doesn't looks correct, it ignore pdc.getFormatPattern() and break the existing PrintDataElement.getValueDisplay(Language) method.